tests/resetdb

```diff
@@ -8,3 +8,7 @@ dropdb -h $1 odin-test || echo "OK, odin-test database hasn't been created befor
 createdb -h $1 odin-test
 
 odin -h $1 -d odin-test include setup.odin
+
+# App figure
+psql -h $1 -d odin-test -c "INSERT INTO odin.app_ledger (reference, app_id, app_name, redirect_url) VALUES ('ref1', 'bowling_app', 'Bowling with Fred', 'http://localhost:8001/odin/app/login/bowling_app/');"
+psql -h $1 -d odin-test -c "INSERT INTO odin.app_owner_ledger (reference, identity_id, app_id) VALUES ('ref1', 'fred', 'bowling_app');"
```

Comment:
What are these commands needed for? Don't we just reset the DB to blank here?

Reply:
I use it just for testing because we don't have a way to set up an app yet so I create the mock app here.